### PR TITLE
templates: add sticky footer

### PR DESCRIPTION
**Pull request by hjhsalo**
* Fix footer scaling issues by introducing a sticky footer
  (closes cernopendata/cernopendata-pages#9)

* Fix footer stacking issues on smaller screens

Signed-off-by: Harri Hirvonsalo <harri.hirvonsalo@cern.ch>

### CODE REVIEW

**Comment by ioannistsanaktsidis** [comment]
Works pretty good, just needs a bit of refactoring in terms of moving scss to a  separate file.

**Comment by tiborsimko** [comment]
@hjhsalo @ioannistsanaktsidis Is this mergable now? It would be good to get all the pending PRs in before we freeze the repositories for the code move.

**Comment by hjhsalo** [comment]
Mergeable, but waiting for review / comments on refactoring:
* SCSS refactored to its own file now.
* Visually OK.
* Rebased on top of current next-branch.

File: cernopendata/modules/theme/static/scss/footer.scss [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+// MA 02111-1307, USA.
+//
+// In applying this license, CERN does not
+// waive the privileges and immunities granted to it by virtue of its status
+// as an Intergovernmental Organization or submit itself to any jurisdiction.
+
+/* From: https://getbootstrap.com/2.3.2/examples/sticky-footer.html*/
+/* Sticky footer styles */
+
+html, body {
```
Comment (ioannistsanaktsidis) [inline cernopendata/modules/theme/static/scss/footer.scss]: I think this one can be moved to the general styles.scss

**Review by ioannistsanaktsidis — CHANGES_REQUESTED** [review]
It looks good to me, only one small change and then we can merge it.

File: cernopendata/modules/theme/static/scss/footer.scss (hunk 1 above)
Comment (hjhsalo) [reply cernopendata/modules/theme/static/scss/footer.scss]: Changed

**Review by ioannistsanaktsidis — APPROVED** [review]

**Pull request merged**